src/indra_cogex/client/enrichment/discrete.py

```diff
@@ -186,7 +186,7 @@ def _do_ora(
 def go_ora(
     client: Neo4jClient,
     gene_ids: Iterable[str],
-    background_gene_ids: Optional[Iterable[str]] = None,
+    background_gene_ids: Optional[Collection[str]] = None,
```

Comment:
Since the annotation for `Iterable` doesn't imply a size, if you happened to pass a generator object, there will be no implementation of `__len__()` and therefore `len()` will raise a TypeError (or some other kind of exception). Therefore, it's better to use `Collection` here since that does imply that there's a size, with all of the other attributes of an `Iterable`.